### PR TITLE
Add an option to whitelist GH organizations

### PR DESCRIPTION
**Pull request by MiLk**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -81,6 +88,26 @@ module.exports = function (kibana) {
...
+                if (allowedOrganizations.indexOf(organization) !== -1) {
+                  request.auth.session.set(request.auth.credentials);
+                  return reply.redirect('./');
+                }
+              }
+              return reply(Boom.forbidden('None of the user\'s organization is allowed'));
+            }).catch(function (error) {
+              return reply(Boom.unauthorized('Unable to verify the user\'s organizations'));
+            });
+            return;
```
Comment (nathanboktae) [inline index.js]: You need to return the promise here so the handler is aware of your async work.

``` javascript
return axios.get(....
```

**Review by nathanboktae — CHANGES_REQUESTED** [review]
Very useful PR. Can you also add documentation and fix the one code issue? Thanks!

File: index.js (hunk 1 above)
Comment (MiLk) [reply index.js]: I'm not familiar with hapi or the kibana plugin ecosystem.
It should be good now.

**Comment by MiLk** [comment]
I've added the documentation.

If you make a release, please be sure to use npm 2 to make the build, as the gulpfile doesn't support npm 3 and the flat structure for dependencies.

The current build for 0.3.0 is not working for me. Dependencies are missing.

Thanks.

**Review by nathanboktae — APPROVED** [review]

**Pull request merged**